keanu-python/examples/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .coal_mining import CoalMining
 from .lorenz_model import LorenzModel
-from .thermometers import Thermometer
\ No newline at end of file
+from . import thermometers
```

Comment:
No. Since `examples` is a package, we need to either include it in an absolute import statement (e.g. `from examples.thermometers` or `from examples import thermometers`) or use an implicit import statement as I did here.

Reply:
ok